packages/themes/src/presets/alternate.css

```diff
@@ -83,7 +83,7 @@
   --scalar-button-1-color: black;
 }
 
-.scalar-api-client__item,
+.scalar-app__item,
```

Comment:
Did this class actually get changed in the source?

Reply:
GREAT CAATCH, yes!